lib/modules/apostrophe-pages/lib/api.js

```diff
@@ -701,28 +701,7 @@ module.exports = function(self, options) {
   //
   // See `moveToTrash` for what the callback receives.
 
-  self.trashInSchema = function(req, _id, callback) {
-    return self.trashInSchemaBody(req, _id, true, callback);
-  };
-
-  // Rescue a page previously trashed via `trashInSchema`. This is an operation that only
-  // makes sense when the `trashInSchema` option flag is set for the module.
-  // Rescues descendants as well. Invokes its callback with `(null, parentSlug, changed)`,
-  // where:
-  //
-  // `parentSlug` is the slug of the parent of the page rescued, for consistency
-  // with the `moveToTrash` method, although the parent does not change;
-  //
-  // `changed` is an array of descendant pages whose trash status also changed,
-  // with `_id` and `slug` properties.
-
-  self.rescueInTree = function(req, _id, callback) {
-    return self.trashInSchemaBody(req, _id, false, callback);
-  };
-
-  // Implementation detail of `trashInSchema` and `rescueInTree`. Accepts a boolean
-  // to do either.
-  self.trashInSchemaBody = function(req, _id, trash, callback) {
+  self.trashInSchema = function(req, _id, toTrash, callback) {
```

Comment:
What's up with these changes to the pages module, I thought you were coming to that later...